### PR TITLE
Update nextant install

### PR DESCRIPTION
**Pull request by Ark74**
Nextant now can be installed, enabled, test & saved configuration from cli.
Tested and works!
Note: Please wait 'til 1.0.4 to commit these changes. Otherwise we'll need to compile the app. (Releases come pre-compiled)

https://github.com/nextcloud/nextant/releases

### CODE REVIEW

**Comment by enoch85** [comment]
Great! :D

File: static/nextant.sh [hunk 1]
```diff
@@ -114,12 +114,12 @@ else
 fi
 rm -r $NT_RELEASE
 sudo -u www-data php $NCPATH/occ app:enable nextant
+sudo -u www-data php $NCPATH/occ nextant:test http://127.0.0.1:8983/solr/ nextant
+sudo -u www-data php $NCPATH/occ nextant:index
 
 if [ $? -eq 0 ]
 then
     echo "Nextant app is now installed and enabled."
```
Comment (enoch85) [inline static/nextant.sh]: Add sleep 3 here

File: static/nextant.sh [hunk 2]
```diff
@@ -114,12 +114,12 @@ else
...
-    echo "Please go to: Admin Settings --> Additional Settings, and configure the app"
-    sleep 5
 else
     echo "Nextant app install failed"
```
Comment (enoch85) [inline static/nextant.sh]: Add sleep 3 here

**Comment by enoch85** [comment]
Released!

**Comment by enoch85** [comment]
Trows an error 

```
nextant enabled
 - Pinging Solr: ok
 - Checking Solr schema: An unhandled exception has been thrown:
Error: Call to a member function getCode() on null in /var/www/nextcloud/apps/nextant/lib/Command/Test.php:144
Stack trace:
#0 /var/www/nextcloud/apps/nextant/lib/Command/Test.php(100): OCA\Nextant\Command\Test->test_schema(Object(Symfony\Component\Console\Output\ConsoleOutput))
#1 /var/www/nextcloud/3rdparty/symfony/console/Command/Command.php(256): OCA\Nextant\Command\Test->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /var/www/nextcloud/core/Command/Base.php(161): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /var/www/nextcloud/3rdparty/symfony/console/Application.php(818): OC\Core\Command\Base->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /var/www/nextcloud/3rdparty/symfony/console/Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(OCA\Nextant\Command\Test), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /var/www/nextcloud/3rdparty/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 /var/www/nextcloud/lib/private/Console/Application.php(169): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 /var/www/nextcloud/console.php(90): OC\Console\Application->run()
#8 /var/www/nextcloud/occ(11): require_once('/var/www/nextcl...')
#9 {main}nextant v1.0.4
Nextant is not yet configured
Nextant app is now installed and enabled.
```

@Ark74 Can you please investigate?

**Comment by Ark74** [comment]
What release are you using?

**Comment by enoch85** [comment]
https://github.com/nextcloud/nextant/issues/137

**Comment by enoch85** [comment]
It always fetches the latest release automaticually. 1,0.4

**Comment by Ark74** [comment]
Mmm.
I'll test with the official latest release i get back to you.

**Comment by Ark74** [comment]
I got caught on an error for the nextant script looking for release 1.0.4 and now we have 1.0.5, this is a special case, but we should take it into account. Since there is no way to prevent the script to keep going until it stop. 
Maybe if an error like a 404 when downloading the package, ask for user to input the release manually or something.
I'll check that again manually.
nt-tag

**Comment by enoch85** [comment]
@Ark74 See https://github.com/nextcloud/vm/pull/129#issuecomment-279133795

**Pull request merged**